src/layout/header/Menu.tsx

```diff
@@ -82,59 +82,64 @@ const Menu = ({ menuButtonLabel, handleLogout }: MenuProps) => {
           vertical: 'bottom',
           horizontal: 'left',
         }}>
-        {user?.isCurator && (
```

Comment:
Aner ikke hvorfor, men plutselig klagde konsollen på dette:
![bilde](https://user-images.githubusercontent.com/6313468/104321798-6349f500-54e4-11eb-8787-0f932ea82e5d.png)
Visste ikke det var mulig å returnere kompnenter i arrays engang, men det er visst fullt mulig, men man må ha `key` hvor det er nestede elementer. Skulle gjerne visst hvorfor det plutslig må gjøres slik nå, for det ble jo brukes fortsatt samme `MuiMenu` 🤔

Reply:
🤔